code/modules/xenoarcheaology/finds/talking.dm

```diff
@@ -34,7 +34,7 @@
 	else if(findtext(msg," ")==0)
 		return
 	else
-		/*var/l = lentext(msg)
+		/*var/l = length(msg)
```

Comment:
Very thorough. :D

Reply:
The someone who left this comment in place will thank me later when they uncomment it and find that it still works and does not generate a deprecation warning :>

I mean, why would it be left in the code in the first place if not for the need to come back later?